tss-esapi/src/context/general_esys_tr.rs

```diff
@@ -388,6 +393,48 @@ impl Context {
         TpmHandle::try_from(tpm_handle)
     }
 
-    // Missing function: Esys_TR_Serialize
-    // Missing function: Esys_TR_Deserialize
+    /// Serialize the metadata of the object identified by `handle` into a new buffer.
+    ///
+    /// This can subsequently be used to recreate the object in a new context in the future.
```

Comment:
That is a good start!

But what I think he meant was that if you look at how the other context methods are documented. They have a description of the arguments to the method, a description of what the method returns, a description of errors that might be thrown (if any) and an example of how to use the method.

Reply:
Thank you, done. :)